tests/api/feed_protocol/test_library_annotator.py

```diff
@@ -914,25 +914,12 @@ def test_active_loan_feed(
             patron=patron,
         )
 
-        response = OPDSAcquisitionFeed.active_loans_for(None, patron, annotator)
+        response = OPDSAcquisitionFeed.active_loans_for(
+            None, patron, annotator
+        ).as_response()
 
         # The feed is private and should not be cached.
         assert isinstance(response, OPDSFeedResponse)
-        assert 0 == response.max_age
-        assert True == response.private
-
-        # Instead, the Last-Modified header is set to the last time
-        # we successfully brought the patron's bookshelf in sync with
-        # the vendor APIs.
-        #
-        # (The timestamps aren't exactly the same because
-        # last_loan_activity_sync is tracked at the millisecond level
-        # and Last-Modified is tracked at the second level.)
-
-        assert response.last_modified is not None
-        assert (
-            patron.last_loan_activity_sync - response.last_modified
-        ).total_seconds() < 1
```

Comment:
Why does this test come out?

Reply:
This code was removed from the annotator, and [added into the controller](https://github.com/ThePalaceProject/circulation/pull/1340/files#diff-944bc0ea08885efcf096971a642422c2f5ea80605fd0b10618c9e6a738d7a837R1470), since the annotator no longer creates the serialized response.
The tests in `tests/api/test_controller_loan.py` already test for that functionality.